google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -988,19 +988,32 @@ public Acl blobToPublicRead(String bucketName, String blobName, long blobGenerat
   }
 
   /** Example of default auth */
-  public Page<Bucket> authListBuckets() {
+  public void authListBuckets() {
     // [START auth_cloud_implicit]
     // If you don't specify credentials when constructing the client, the
     // client library will look for credentials in the environment.
 
     Storage storage = StorageOptions.getDefaultInstance().getService();
 
     Page<Bucket> buckets = storage.list();
+    System.out.println("Buckets:");
     for (Bucket bucket : buckets.iterateAll()) {
-      // do something with the info
+      System.out.println(bucket.getName());
     }
     // [END auth_cloud_implicit]
-    return buckets;
+  }
+
+  /** Example of a simple listBuckets() */
```

Comment:
Am I missing something? These two methods look exactly the same aside from the name.